README.md

```diff
@@ -85,14 +84,14 @@ automatically optimize it for production-ready environments.
 
 ## Getting started
 
-Towhee requires Python 3.6+. Towhee can be installed via `pip`:
+Towhee requires Python 3.6+. You can install Towhee via `pip`:
 
 ```bash
 % pip install -U pip  # if you run into installation issues, try updating pip
 % pip install towhee
 ```
 
-Try your first Towhee pipeline. In this example, we show how to create a CLIP-based cross modal retrieval pipeline within 15 lines of code.
+Try your first Towhee pipeline. For example, create a CLIP-based cross modal retrieval pipeline within 15 lines of code.
```

Comment:
Try your first Towhee pipeline. For example, create a CLIP-based cross modal retrieval pipeline with 15 lines of code.